paybylink-example/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@using adyen_dotnet_paybylink_example.Models;
+@model List<PaymentLinkModel>
+
+@{
+    ViewData["Title"] = "Adyen Pay By Link Home";
+}
+
+<div class="main-container">
+  <div class="info-mainsection">
+    <h1>Pay By Link Demo</h1>
+    <p>To use this interactive example of Pay By Link, provide an <b>amount</b> in minor units (e.g. 1000 = 10 EUR) and <b>reference</b> (guid). Set <b>isReusable</b> to <i>true</i> to accept multiple payments per payment link.</p>
```

Comment:
It would be better not to use minor units (as we wouldn't ask a shopper to do so)

Reply:
I've created a ticket for this change, as it's not as trivial to multiple by 100 (it makes sense for EUR), but for a better approach, we should implement a helper function in the libraries: https://docs.adyen.com/development-resources/currency-codes

Reply:
Let's showcase the best practices here ^ I've left this open in minor-units (for now) and created a ticket to address this issue